### ISSUE TITLE
trainretriever 

### ISSUE THREAD

**Issue by chenzhongwu** [texttron/tevatron#116]
Hi! Should I write the code for generating the training datasets for the retriever by myself? 
What if my own datasets are hard to choose positive and negative passages?
Thanks!

**Comment by MXueguang** [comment]
yes, for each query, the training requires at least a positive passages (human judged or heuristically selected.)

**Comment by chenzhongwu** [comment]
In the search part in example_dpr.md, there should be a tevatron.faiss_retriever, but I did not find it. Did you miss it? Could you add tevatron.faiss_retriever?
Thanks!
